galgebra/mv.py

```diff
@@ -177,8 +177,7 @@ def characterise_Mv(self):
         return
 
     def make_grade(self, *args, **kwargs):
-    # Called by __init__ to make a pure grade multivector.
-
+        # Called by __init__ to make a pure grade multivector.
```

Comment:
github struggles to diff with these unindented comments

Reply:
Viewing the diff in [ignore-whitespace mode (`?w=1`)](https://github.com/pygae/galgebra/pull/58/files?w=1) helps.